libs/partners/upstage/langchain_upstage/embeddings.py

```diff
@@ -1,34 +1,242 @@
-from typing import List
+import logging
+import os
+import warnings
+from typing import (
+    Any,
+    Dict,
+    List,
+    Literal,
+    Mapping,
+    Optional,
+    Sequence,
+    Set,
+    Tuple,
+    Union,
+)
 
+import openai
 from langchain_core.embeddings import Embeddings
+from langchain_core.pydantic_v1 import (
+    BaseModel,
+    Extra,
+    Field,
+    SecretStr,
+    root_validator,
+)
+from langchain_core.utils import (
+    convert_to_secret_str,
+    get_from_dict_or_env,
+    get_pydantic_field_names,
+)
 
+logger = logging.getLogger(__name__)
 
-class UpstageEmbeddings(Embeddings):
+
+class UpstageEmbeddings(BaseModel, Embeddings):
     """UpstageEmbeddings embedding model.
 
+    To use, set the environment variable `UPSTAGE_API_KEY` with your API key or
+    pass it as a named parameter to the constructor.
+
     Example:
         .. code-block:: python
 
             from langchain_upstage import UpstageEmbeddings
 
-            model = UpstageEmbeddings()
+            model = UpstageEmbeddings(model="solar-1-mini-embedding-query")
```

Comment:
```suggestion
            model = UpstageEmbeddings()
```

여기도 가장 단순한 모드로 이렇게 가면 좋을것 같아요!